src/components/EarningsTable/EarningsTable.js

```diff
@@ -140,7 +131,7 @@ function EarningsTable() {
     // filter out keys we don't want to submit
     Object.keys(filtersToSubmit).forEach((k) => {
       if (
-        filtersToSubmit[k] === 'all' ||
+        filtersToSubmit[k].includes('all') ||
```

Comment:
Unfortunately this causes the opposite behaviour to what we want.
User should be able to type "all" into the filter and see only results containing the string "all".
We need another way to represent _show all_ in `filtersToSubmit`.

Reply:
@Hengxin-Liu Are you still working on this issue?